src/Microsoft.Azure.WebJobs.Host/Bindings/BindingExpressionAttribute.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+using Microsoft.Azure.WebJobs.Host.Config;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Attribute to allow binding directly to binding data. 
+    /// </summary>
+    [Binding]
+    public class BindingExpressionAttribute : Attribute
```

Comment:
I think this attribute should be defined in the core project (peer to QueueAttribute, etc.)